server/ctools_direct_response.rb

```diff
@@ -91,14 +91,24 @@ def self.filter(assignment)
 
     logger.debug "#{self.class.to_s}:#{__method__}:#{__LINE__}: assignment: [#{assignment.inspect}]"
     calendarItem = assignment['calendarItem']
-    return nil if calendarItem.nil?
+
+    if calendarItem.nil?
+      logger.debug "#{self.class.to_s}:#{__method__}:#{__LINE__}: assignment: fail for nil calendarItem"
```

Comment:
The entire assignment is logged in an earlier debug statement, so it doesn't need to be repeated here.  That would be a good idea if/when the earlier log statement is taken out.